redaxo/src/core/lib/util/socket/socket.php

```diff
@@ -110,6 +118,20 @@ public function setPath($path)
         return $this;
     }
 
+    /**
+     * Sets the socket options.
+     *
+     * @param array $options
+     *
+     * @return $this Current socket
+     */
+    public function setOptions($options = [])
```

Comment:
```suggestion
    public function setOptions(array $options)
```

Wenn man die Methode aufruft, sollte man auch immer einen Wert übergeben, daher Default-Wert weg.

Reply:
gute HInweise .. ja